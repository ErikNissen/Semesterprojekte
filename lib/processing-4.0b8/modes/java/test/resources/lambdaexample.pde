version https://git-lfs.github.com/spec/v1
oid sha256:fe4888de7c7946cf6b065be212177b08aa0bedb6afad05b85330eef64cfdb028
size 128
