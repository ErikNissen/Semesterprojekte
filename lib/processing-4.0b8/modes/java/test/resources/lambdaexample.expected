version https://git-lfs.github.com/spec/v1
oid sha256:2aca13265e0dd05bcd2e87ffeec8be32d638f7839d1fea1cda30da7d41beacec
size 828
